README.md

```diff
@@ -13,22 +13,6 @@ As this charm is not yet published, you need to follow the build and deploy inst
 - `reload-pgbouncer`
   - Reloads the pgbouncer application.
   - Currently only implemented as a stub.
-- `add-user`
-  - Adds a new pgbouncer user.
-  - Params:
-    - username
-    - password
-- `remove-user`
-  - Removes an existing pgbouncer user.
-  - Params:
-    - username
-- `change-password`
-  - Updates the password for a given pgbouncer user.
-  - Params:
-    - username
-    - password
-- `get-users`
-  - Lists the existing pgbouncer users.
```

Comment:
You kind of elluded to this in your description, but why are leftover actions being removed?

Reply:
The handlers for these actions were removed in the last PR because this charm should be accessing postgres users from its relation to the postgres charm.